docs/content/modules/tensor_mechanics/systems/Bounds/RankTwoScalarAux.md

```diff
@@ -1,11 +0,0 @@
-<!-- MOOSE Object Documentation Stub: Remove this when content is added. -->
-#RankTwoScalarAux
-!devel /Bounds/RankTwoScalarAux float=right width=auto margin=20px padding=20px background-color=#F8F8F8
-
-!description /Bounds/RankTwoScalarAux
-
-!parameters /Bounds/RankTwoScalarAux
-
-!inputfiles /Bounds/RankTwoScalarAux
-
-!childobjects /Bounds/RankTwoScalarAux
```

Comment:
Bounds was an identical copy of the Auxkernels folder, so I removed bounds since auxkernels matches the code structure